config/kubernetes/sig-docs/teams.yaml

```diff
@@ -173,14 +185,15 @@ teams:
     - jcjesus # Portuguese
     - markthink # Chinese
     - micheleberardi # Italian
-    - mittalyashu #hindi
+    - mittalyashu # Hindi
     - nasa9084 # Japanese
     - raelga # Spanish
     - remyleone # French
     - rlenferink # Italian
     - tnir # Japanese
     - zacharysarah # English
     - zhangxiaoyu-zidif # Chinese
+    - kittenking # Tamil
```

Comment:
`config_test.go:250: The team sig-docs-l10n-admins in org Kubernetes has an unsorted list of members`

Reply:
@cblecker Sorted now!